templates/multisig/contracts/FundraisingMultisigTemplate.sol

```diff
@@ -0,0 +1,474 @@
+pragma solidity 0.4.24;
+
+import "@aragon/apps-agent/contracts/Agent.sol";
+import "@aragon/templates-shared/contracts/BaseTemplate.sol";
+import "@ablack/fundraising-bancor-formula/contracts/BancorFormula.sol";
+import "@ablack/fundraising-tap/contracts/Tap.sol";
+import {AragonFundraisingController as Controller} from "@ablack/fundraising-aragon-fundraising/contracts/AragonFundraisingController.sol";
+import {BatchedBancorMarketMaker as MarketMaker} from "@ablack/fundraising-batched-bancor-market-maker/contracts/BatchedBancorMarketMaker.sol";
+
+
+contract FundraisingMultisigTemplate is BaseTemplate {
+    string constant private ERROR_EMPTY_BOARD = "FM_EMPTY_BOARD";
+    string constant private ERROR_BAD_VOTE_SETTINGS = "FM_BAD_VOTE_SETTINGS";
+    string constant private ERROR_BAD_FUNDRAISING_SETTINGS = "FM_BAD_FUNDRAISING_SETTINGS";
+    string constant private ERROR_MISSING_CACHE = "FM_MISSING_CACHE";
+
+    bool    constant private BOARD_TRANSFERABLE = false;
+    uint8   constant private BOARD_TOKEN_DECIMALS = uint8(0);
+    uint256 constant private BOARD_MAX_PER_ACCOUNT = uint256(1);
+
+    bool    constant private SHARE_TRANSFERABLE = true;
+    uint8   constant private SHARE_TOKEN_DECIMALS = uint8(18);
+    uint256 constant private SHARE_MAX_PER_ACCOUNT = uint256(0);
+
+    uint64 constant private SHARE_VOTE_DURATION = uint64(1 weeks);
+    uint64 constant private SHARE_SUPPORT_REQUIRED = uint64(50 * 10**16);
+    uint64 constant private SHARE_MIN_ACCEPTANCE_QUORUM = uint64(0);
+    uint64[3]       private SHARE_VOTING_SETTINGS = [SHARE_SUPPORT_REQUIRED, SHARE_MIN_ACCEPTANCE_QUORUM, SHARE_VOTE_DURATION];
```

Comment:
this is not used in the code - which is good because it should not be hardcoded.